### PR TITLE
Answers chp 11

### PR DESCRIPTION
**Pull request by AlexAndorra**
A first version of chapter 11 solutions. 

As discussed with @aloctavodia **this is not ready to merge yet**, mainly because of sampling problems in ordered categorical model (problem 11H5) and a question about posterior predictive samples (see end of problem 11H6). 

Will do the necessary updates after review.
Thanks in advance, 
PyMCheers !

### CODE REVIEW

File: Rethinking/Data/fish.csv [hunk 1]
```diff
@@ -0,0 +1,501 @@
+fish_caught;livebait;camper;persons;child;hours
```
[the remark this answers is not in the document]
Comment (AlexAndorra) [reply Rethinking/Data/fish.csv]: Done ;)

File: Rethinking/Data/hurricanes.csv [hunk 2]
```diff
@@ -0,0 +1,185 @@
+"name";"year";"deaths";"category";"min_pressure";"damage_norm";"female";"femininity"
```
[the remark this answers is not in the document]
Comment (AlexAndorra) [reply Rethinking/Data/hurricanes.csv]: Done ;)

**Comment by aloctavodia** [comment]
I think we can merge if we open and issue to remind us to revisit the ordered categorical model in problem 11H5. What do you think @AlexAndorra?

**Comment by AlexAndorra** [comment]
Agreed Osvaldo! Let me adjust and push the NB in that sense tomorrow. I'll
open the issue right after that 😉

I think we can do the same thing for chapter 12 (ready to merge, but last 2
problems need Ordered Logistic) if it's fine by you?

Le mer. 3 juil. 2019 à 17:22, Osvaldo Martin <notifications@github.com> a
écrit :

**Comment by aloctavodia** [comment]
Totally fine! Thanks.

**Comment by AlexAndorra** [comment]
Ready for merge and opened the issue as a reminder, as discussed :)

**Pull request merged**